src/main/java/seedu/application/logic/commands/CommandResult.java

```diff
@@ -48,6 +51,10 @@ public boolean isExit() {
         return exit;
     }
 
+    public Index isInterview() {
```

Comment:
Maybe consider if this is the most apt method name since it sounds like a boolean instead of returning the index